test/SparqlAntlrParserTest.cpp

```diff
@@ -661,3 +661,120 @@ TEST(SparqlParser, Bind) {
     EXPECT_THAT(bindAndText._remainingText, IsEmpty());
   }
 }
+
+TEST(SparqlParser, Integer) {
+  {
+    string input = "1931";
+    ParserAndVisitor p{input};
+
+    unsigned long long result =
+        p.parser.integer()->accept(&p.visitor).as<unsigned long long>();
+    EXPECT_EQ(result, 1931ULL);
+  }
+
+  {
+    string input = "0";
+    ParserAndVisitor p{input};
+
+    unsigned long long result =
+        p.parser.integer()->accept(&p.visitor).as<unsigned long long>();
+    EXPECT_EQ(result, 0ULL);
```

Comment:
In QLever the style is `ull` instead of `ULL` (both are possible, but the goal is to make the codebase more consistent as it grows). [Several occurences below and above].